docs/user-guide/src/introduction.md

```diff
@@ -0,0 +1,28 @@
+<div style="float: right; position: relative; display: inline;">
+    <img src="images/metal3-color.svg" width="160px" />
+</div>
+
+# Metal³
+
+Metal³ project (pronounced: Metal Kubed) exists to provide components
+that allow you to do bare metal host management for Kubernetes. Metal³
+works as a Kubernetes application, meaning it runs on Kubernetes and is
+managed through Kubernetes interfaces.
+
+Metal³ is one of the providers for the Kubernetes sub-project [Cluster
+API](https://github.com/kubernetes-sigs/cluster-api) and is focused on
+providing Kubernetes native APIs to manage Kubernetes clusters in a bare
+metal environment.
+
+## Why another provisioning tool
```

Comment:
We could add some context here as well, maybe?

Reply:
Yes, I can remove it. It was just some initial text to get started with the mdbook and see how it works.